### PR TITLE
Extract nes_font_static

### PR DESCRIPTION
**Pull request by EllipticEllipsis**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

Extracted nes_font_static, using the same format as OoT: this has the advantage that the character numbers line up with their values in the messages, and that if you have your file browser set up correctly, they will display in the right order.

I would ask that reviewers test this themselves and look carefully at all the characters: it's quite easy to mess this up accidentally: I looked myself, but I'm sure I missed something.

### CODE REVIEW

File: assets/xml/interface/nes_font_static.xml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        <Texture Name="gMsgChar28LeftParenthesesTex" OutName="msg_char_28_left_parentheses" Format="i4" Width="16" Height="16" Offset="0x400"/>
+        <Texture Name="gMsgChar29RightParenthesesTex" OutName="msg_char_29_right_parentheses" Format="i4" Width="16" Height="16" Offset="0x480"/>
+        <Texture Name="gMsgChar2AAsteriskTex" OutName="msg_char_2a_asterisk" Format="i4" Width="16" Height="16" Offset="0x500"/>
+        <Texture Name="gMsgChar2BPlusSignTex" OutName="msg_char_2b_plus_sign" Format="i4" Width="16" Height="16" Offset="0x580"/>
+        <Texture Name="gMsgChar2CCommaTex" OutName="msg_char_2c_comma" Format="i4" Width="16" Height="16" Offset="0x600"/>
+        <Texture Name="gMsgChar2DHyphenMinusTex" OutName="msg_char_2d_hyphen_minus" Format="i4" Width="16" Height="16" Offset="0x680"/>
+        <Texture Name="gMsgChar2EFullStopTex" OutName="msg_char_2e_full_stop" Format="i4" Width="16" Height="16" Offset="0x700"/>
+        <Texture Name="gMsgChar2FSolidusTex" OutName="msg_char_2f_solidus" Format="i4" Width="16" Height="16" Offset="0x780"/>
```
Comment (AngheloAlf) [inline assets/xml/interface/nes_font_static.xml:21]: Why "solidus" and not "slash"?

File: assets/xml/interface/nes_font_static.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        <Texture Name="gMsgChar58LatinCapitalLetterXTex" OutName="msg_char_58_latin_capital_letter_x" Format="i4" Width="16" Height="16" Offset="0x1C00"/>
+        <Texture Name="gMsgChar59LatinCapitalLetterYTex" OutName="msg_char_59_latin_capital_letter_y" Format="i4" Width="16" Height="16" Offset="0x1C80"/>
+        <Texture Name="gMsgChar5ALatinCapitalLetterZTex" OutName="msg_char_5a_latin_capital_letter_z" Format="i4" Width="16" Height="16" Offset="0x1D00"/>
+        <Texture Name="gMsgChar5BLeftSquareBracketTex" OutName="msg_char_5b_left_square_bracket" Format="i4" Width="16" Height="16" Offset="0x1D80"/>
+        <Texture Name="gMsgChar5CYenSignTex" OutName="msg_char_5c_yen_sign" Format="i4" Width="16" Height="16" Offset="0x1E00"/>
+        <Texture Name="gMsgChar5DRightSquareBracketTex" OutName="msg_char_5d_right_square_bracket" Format="i4" Width="16" Height="16" Offset="0x1E80"/>
+        <Texture Name="gMsgChar5ECircumflexAccentTex" OutName="msg_char_5e_circumflex_accent" Format="i4" Width="16" Height="16" Offset="0x1F00"/>
+        <Texture Name="gMsgChar5FLowLineTex" OutName="msg_char_5f_low_line" Format="i4" Width="16" Height="16" Offset="0x1F80"/>
```
Comment (AngheloAlf) [inline assets/xml/interface/nes_font_static.xml:69]: Why "low line" instead of "underscore"?

File: assets/xml/interface/nes_font_static.xml (hunk 1 above)
Comment (EllipticEllipsis) [reply assets/xml/interface/nes_font_static.xml:21]: https://unicode-table.com/en/002F/

File: assets/xml/interface/nes_font_static.xml (hunk 2 above)
Comment (EllipticEllipsis) [reply assets/xml/interface/nes_font_static.xml:69]: https://unicode-explorer.com/c/005F

**Review by AngheloAlf — APPROVED** [review]

**Comment by EllipticEllipsis** [comment]
I'm not saying they're good names ("hyphen minus" certainly isn't), but at least they're standard.

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]
Not sure that we want the map code to be in the texture name for each character, but I don't feel strongly.

**Pull request merged**